File: JAVA/Figures exercise/com/figures/Rectangle.java
package com.figures;

public class Rectangle extends FigureGeometric {
    private int base;
    private int height;
    private double calculate;

    public  Rectangle (int base, int height) {
        super("Rectangle", "Rectangulo");
        this.base = base;
        this.height= height;
    }
    public int getBase() {
        return this.base;
    }
    public int getHeight(){
        return this.height;
    }
    public double calculateArea() {
        this.calculate = this.getBase() * this.getHeight();
        return this.calculate;
    }
    
}

File: PHP/Figures exercise/figures/rectangle.php
<?php
 class Rectangle extends FigureGeometric {

   private float $base;
   private float $height;
   private float $calculate;
   
   public function __construct(float $base, float $height){
       parent::_construct("Rectangle");
       $this-> base= $base;
       $this->height=$height;
   }
   
   public function getBase():float{
       return $this-> base;
   }
   public function getHeight():float{
       return $this-> height;
   }
   public function calculateArea():float{
       $this->calculate = $this-> getBase()* $this->getHeight();
       return $this->calculate;
   }
}